node/lib/packaging/PackageInfo.ts

```diff
@@ -0,0 +1,173 @@
+
+/**
+ * Information about an OpenT2T package, as loaded from the package.json.
+ */
+export class PackageInfo {
```

Comment:
It's here already below as `PackageTranslatorInfo.onboarding`

I expect there will also need to be a `PackageOnboardingInfo` class added here when we fill out the onboarding design.

---
In reply to: [73275191](https://github.com/openT2T/opent2t/pull/2#discussion_r73275191) [](ancestors = 73275191)